chess/events/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@chess/events",
+  "version": "1.0.0",
+  "description": "Events for Chess game, to be imported into any frontend",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc -p tsconfig.json && tsc-alias -p tsconfig.json",
+    "lint": "prettier --write '**/*.ts'"
+  },
+  "keywords": [],
+  "author": "PaimaStudios",
+  "dependencies": {
+    "esrun": "^3.2.26",
+    "mqtt": "5.8.1",
+    "@sinclair/typebox": "0.33.4"
```

Comment:
If types need to be defined with `typebox` perhaps it should be exposed in paima-engine